p7_integration_tests/scripts/synfire_run.py

```diff
@@ -0,0 +1,195 @@
+"""
+Synfirechain-like example
+"""
+try:
+    import pyNN.spiNNaker as p
+except Exception as e:
+    import spynnaker.pyNN as p
+from spynnaker.pyNN.models.neuron.builds.if_curr_exp \
+    import IFCurrExp as IF_curr_exp
+
+CELL_PARAMS_LIF = {'cm': 0.25, 'i_offset': 0.0, 'tau_m': 20.0,
+                   'tau_refrac': 2.0, 'tau_syn_E': 5.0, 'tau_syn_I': 5.0,
+                   'v_reset': -70.0, 'v_rest': -65.0, 'v_thresh': -50.0}
+
+
+def do_run(nNeurons, timestep=1, max_delay=144.0, spike_times=[[0]],
```

Comment:
spike_times=[[0]] opens up issues with python. Its not mutable for all future versions. So say run 1 appended [1] to the list. on the second run though, spikes times = [[0],[1]] 

its a gotcha which took us a while to track down, it might not be used in this way here, but id rather be safe than sorry and have spike_times=None and a check inside to set None to [[0]]

The same is said for runtimes